immuneML/encodings/text_as_int_encoding/TextAsIntEncoder.py

```diff
@@ -0,0 +1,78 @@
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+
+from immuneML.caching.CacheHandler import CacheHandler
+from immuneML.data_model.dataset.SequenceDataset import SequenceDataset
+from immuneML.data_model.encoded_data.EncodedData import EncodedData
+from immuneML.encodings.DatasetEncoder import DatasetEncoder
+from immuneML.encodings.EncoderParams import EncoderParams
+from immuneML.util.PathBuilder import PathBuilder
+
+
+class TextAsIntEncoder(DatasetEncoder):
```

Comment:
what is the difference between TextAsIntEncoder and CharToIntEncoder? Also, documentation? :)